src/AutoRest.CSharp.V3/JsonRpc/StandaloneAutoRestInterface.cs

```diff
@@ -24,11 +24,14 @@ public StandaloneAutoRestInterface(string[] args)
                 {
                     name = name.Substring(2);
                 }
-                _arguments[name] = parts[1];
                 if (parts.Length == 1)
                 {
                     _arguments[name] = "true";
                 }
+                else
+                {
+                    _arguments[name] = parts[1];
```

Comment:
If it will always be set, why not:
```csharp
_arguments[name] =  parts.Length == 1 ? "true" : parts[1];
```
Elses tend to make intent less clear and makes readability more cumbersome.

Reply:
Good point, I'll change in the following PR for the query.